code/game/objects/items/toys.dm

```diff
@@ -1000,7 +1000,7 @@
 
 /obj/item/toy/gasha/snowflake
 	name = "toy snowflake"
-	desc = "What a faggot."
+	desc = "What a unique and interesting toy, unlike all the others!"
```

Comment:
Honestly this item should just be removed since it's a reference to a (former) player, but IIRC the person to whom it is a reference was the one who wrote this. Not that the mod team would care. Also, see one line below

Reply:
it's on three maps so I'd rather not remove it